src/utils/collapsedRow/index.ts

```diff
@@ -0,0 +1,12 @@
+import { ProcessedLogLine } from "types/logs";
+
+/**
+ * isCollapsedRow determines if a row is a collapsed row
+ * although it is a simple function its purpose is to make the code more readable
+ * @param logLine ProcessedLogLine
+ * @returns boolean true if the row is a collapsed row
+ */
+const isCollapsedRow = (logLine: ProcessedLogLine): logLine is number[] =>
```

Comment:
This fancy `logline is number[]` syntax supports typescripts control flow analysis